Wikipedia/Code/NotificationsCenterCell.swift

```diff
@@ -337,12 +337,13 @@ final class NotificationsCenterCell: UICollectionViewCell {
         projectSourceContainer.addSubview(projectSourceLabel)
         projectSourceContainer.addSubview(projectSourceImage)
 
+        let minimumSummaryHeight = (traitCollection.horizontalSizeClass == .regular) ? 40.0 : 64.0
```

Comment:
One issue I've noticed is that the 2nd label gets squished when are on Notifications Center and change iPad to split screen (see screenshot), but it works when loading this screen fresh while in a split screen.

![IMG_1243](https://user-images.githubusercontent.com/3620196/161118901-3ed4824a-1409-47ce-bcc6-75bec055e622.PNG)

I think this is related to our cell height caching. I was able to fix it by making this change [here](https://github.com/wikimedia/wikipedia-ios/blob/40291b2d8b28d4cb35537c93675e1ad3d2972e6d/Wikipedia/Code/NotificationsCenterView.swift#L74):

```
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
        super.traitCollectionDidChange(previousTraitCollection)

        if previousTraitCollection?.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory {
            emptyOverlayHeaderLabel.font = UIFont.wmf_font(.mediumBody, compatibleWithTraitCollection: traitCollection)
            emptyOverlaySubheaderLabel.font = UIFont.wmf_font(.subheadline, compatibleWithTraitCollection: traitCollection)
        }
        
        if previousTraitCollection?.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory || previousTraitCollection?.horizontalSizeClass != traitCollection.horizontalSizeClass {
            calculatedCellHeight = nil
        }
    }
```

Reply:
Done